tests/docker_noinit/Dockerfile

```
@@ -22,6 +22,8 @@ RUN apk add --no-cache \
 # symlink go-task to task
 RUN ln -s /usr/bin/go-task /usr/local/bin/task
 
+RUN npm install -g turbo
```

Hadolint (2.14.0)

[warning] Pin versions in npm. Instead of `npm install <package>` use `npm install <package>@<version>`

(DL3016)